### PR TITLE
Assignmen01 @hgill17

### PR DESCRIPTION
**Pull request by dcox2005**
https://icy-pond-058a8a210.1.azurestaticapps.net/

updated to new web page that is different from main on that class is running on

### CODE REVIEW

File: wordle-web/pages/game.vue [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        <v-col class="text-right">
+          <v-btn color="purple" href="./">
+            Return to main page
+          </v-btn>
+        </v-col>
+        <div>
+            <v-btn color="blue" href="https://qntm.org/files/absurdle/absurdle.html" target="_blank">
+            Try another game
+          </v-btn>
+          </div>
```
Comment (Tom-Rohr) [inline wordle-web/pages/game.vue:46]: minor formatting error

File: wordle-web/pages/game.vue [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    getWord()
+    {
+        this.getWordButtonText = this.getWordButtonText === 'Get Your Word' ? 'Word Retrieved' : 'Get Your Word';
+        this.getWordLoading = true;
+        setTimeout(() => {
+            this.getWordLoading = false;
+        }, 1000);
+    }
+}
+</script>
```
Comment (Tom-Rohr) [inline wordle-web/pages/game.vue:84]: /game page looks well organized.

File: wordle-web/pages/index.vue [hunk 3]
```diff
@@ -73,6 +73,7 @@
         </v-card-text>
         <v-card-actions>
           <v-spacer />
+          <v-btn color="purple" href="./game">GO TO GAME</v-btn>
```
Comment (Tom-Rohr) [inline wordle-web/pages/index.vue:76]: Button works. Excellent color choice.

**Comment by dcox2005** [comment]
updated to a new web page that is different from the main one that class code is running on.

https://icy-pond-058a8a210.1.azurestaticapps.net/

File: wordle-web/pages/game.vue [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+          1)Your guess has to be a real word. No random letters.<br>
+          2)Gray letters are not found in the word at all.<br>
+          3)Yellow letter are correct guesses but in the wrong spot.<br>
+          4)Green letters are correct guesses in the correct spot. Don't move them.<br>
+          5)Words can have multiples of the same letter, ex: Apple<br>
+          6)You have {{maxTries}} tries to guess the correct word.<br>
+        </p>
+        <p class="display-1">
+          Now let's play a game!
+        </p>
```
Comment (mmwoodfo) [inline wordle-web/pages/game.vue:22]: No need to wrap these in `<p>` tags. Instead, you should wrap them in their on individual `<v-card-text>` tags like so:
```suggestion
        <v-card-text class="headline">
          I want to play a game!
        </v-card-text>
        <v-card-text class="body-1">
          Here are the rules:<br>
          1)Your guess has to be a real word. No random letters.<br>
          2)Gray letters are not found in the word at all.<br>
          3)Yellow letter are correct guesses but in the wrong spot.<br>
          4)Green letters are correct guesses in the correct spot. Don't move them.<br>
          5)Words can have multiples of the same letter, ex: Apple<br>
          6)You have {{maxTries}} tries to guess the correct word.<br>
        </v-card-text>
        <v-card-text class="display-1">
          Now let's play a game!
        </v-card-text>
```

File: wordle-web/pages/inspire.vue [hunk 5]
```diff
@@ -8,6 +8,11 @@
           <small>
             <em>&mdash;John Johnson</em>
           </small>
+          <div>
+            <v-btn color="purple" href="./">
```
[the remark this answers is not in the document]
Comment (mmwoodfo) [reply wordle-web/pages/inspire.vue:12]: So the benefit is that one supports SPA and one doesn't. `to="/'` will not reload everything, only the components on the page that is changing. You'll notice when using `href="./"` the entire page gets reloaded.

So, you should be using the `to="/"` to change to your home page.

**Review by mmwoodfo — COMMENTED** [review]
Great job! Good job using an Assignment 1 branch too, rather than keeping everything on main. Great job at git branch management. :)

**Pull request closed without merging**